src/main/java/uk/gov/hmcts/reform/pip/data/management/errorhandling/exceptions/HeaderValidationException.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.pip.data.management.errorhandling.exceptions;
+
+
+/**
+ * Custom exception which handles all header validation issues.
+ */
+
+public class HeaderValidationException extends RuntimeException {
```

Comment:
```suggestion
public class HeaderValidationException extends ValidationException {
```

Reply:
Updated this now

Reply:
i cant see the update on this

Reply:
I've realised I did update this originally, but then changed it back when sonarqube flagged a few issues as a code smell.

See response below